phpstan-baseline.neon

```diff
@@ -0,0 +1,21 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Function args\\\\args\\(\\) has parameter \\$word with no type specified\\.$#"
```

Comment:
Why is this argument typing not fixed directly?

Reply:
It's done on purpose :)